cmd/pbapi/main.go

```diff
@@ -103,9 +101,9 @@ func main() {
 	if err != nil {
 		log.Fatal().Err(err).Msg("Error initializing dejq queue")
 	}
+	dejq.RegisterJobs(&logger)
 	if config.Worker.Queue == "memory" {
```

Comment:
We have to register jobs now even in pbapi because registering also performs interface type registration for gob encoder.